lib/v2/polkadot/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'polkadot.network': {
+        _name: 'Polkadot',
+        www: [
+            {
+                title: 'blog',
+                docs: 'https://docs.rsshub.app/blog.html#polkadot',
+                source: ['/', '/blog/', '/blog/:name/'],
```

Comment:
Maybe you want a wildcard match instead.
```suggestion
                source: ['/', '/blog/', '/blog/*'],
```